packages/lesswrong/server/callbacks/userJobAdCallbacks.tsx

```diff
@@ -0,0 +1,33 @@
+import { getCollectionHooks } from '../mutationCallbacks';
+import { wrapAndSendEmail } from '../emails/renderEmail';
+import '../emailComponents/EmailJobAdReminder';
+import React from 'react';
+import { Components } from '../../lib/vulcan-lib';
+import { JOB_AD_DATA } from '../../components/ea-forum/TargetedJobAd';
+import moment from 'moment';
+
+getCollectionHooks("UserJobAds").updateAsync.add(async function setJobAdReminderEmail(
+  {document, oldDocument, currentUser}: {document: DbUserJobAd, oldDocument: DbUserJobAd, currentUser: DbUser|null}
+) {
+  // Skip if we can't email this user
+  if (!currentUser || !currentUser.email || currentUser.deleted) return
```

Comment:
Maybe we should check `user.unsubscribeFromAll` here too (the description in the schema is "Do not send me any emails").

Reply:
Oh, I just saw the line later on where we force this if the user checked "unsubscribe from all". I don't know - it seems a bit sketchy to me when they checked "Do not send me any emails". Can we send an onsite notification as well/instead?